File: responses/S8MADQ-BEC/1_sum_of_integers_in_a_string.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A):
    # write your code in Python
    if not len(A):
        return 0
    
    mx=-2**32
    count=0
    
    for it in A:
        if it>mx:
            count=1
            mx=it
        elif it==mx:
            count+=1
    
    return count

File: responses/MMVQP9-WAB/1_sum_of_integers_in_a_string.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(S) {
    // write your code in JavaScript
    var lst = S.split(",");
    var prev_value = 0;
    for (i = 0;  i < lst.length; i++) {
        prev_value = parseInt(lst[i]) + prev_value;
    }
    return prev_value;
}